sdk/netapp/azure-mgmt-netapp/tests/test_snapshot_policies.py

```diff
@@ -54,10 +53,11 @@ class TestNetAppSnapshotPolicy(AzureMgmtRecordedTestCase):
     def setup_method(self, method):
         self.client = self.create_mgmt_client(azure.mgmt.netapp.NetAppManagementClient)
         if self.is_live:
-            self.network_client.virtual_networks.begin_delete(TEST_RG, VNETNAME)
+            from azure.mgmt.network import NetworkManagementClient
+            self.network_client = self.create_mgmt_client(NetworkManagementClient)
 
 
-    # Before tests are run live a resource group needs to be created along with vnet and subnet
+            # Before tests are run live a resource group needs to be created along with vnet and subnet
```

Comment:
@hildurhodd Why do you add the blank space

Reply:
This was of course a mistake; I have already fixed it.